BackofficeBundle/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -1,5 +1,18 @@
 {% extends 'BraincraftedBootstrapBundle:Form:bootstrap.html.twig' %}
 
+{% block form %}
```

Comment:
no, in the `form_start` block, it will only add a class `has-global-errors` to the form. But this class is added only if they are form related errors. In my case, there is only a field error.